routes/tokens.js

```diff
@@ -0,0 +1,158 @@
+'use strict';
+
+var Joi = require('joi');
+var Boom = require('boom');
+var ObjectId = require('mongodb').ObjectID;
+var crypto = require('crypto');
+var _ = require('lodash');
+
+module.exports = [
+  {
+    method: 'GET',
+    path: '/tokens',
+    config: {
+      auth: 'session'
+    },
+    handler: function (request, reply) {
+      var tokensCol = request.server.plugins.db.connection.collection('tokens');
+
+      tokensCol.find().toArray(function (err, tokens) {
+        if (err) {
+          return reply(Boom.badImplementation());
```

Comment:
I think it's better to send Boom.wrap(err) here, so that we'll more useful message back from the request and in the logs.